utonics/labproject/main.go

```diff
@@ -266,16 +266,20 @@ func newProject(values map[string][]string, botClient, userClient *worker.Client
 		return msgs, err
 	}
 
-	for _, submodule := range submodules {
+	for smName, submodule := range submodules {
 		os.Chdir(submodule.path)
+		smName = strings.ReplaceAll(smName, "/", "_")
 		msgs = append(msgs, "Uploading submodule to new project repository")
-		if err := uploadProjectRepository(botClient, remoteName); err != nil {
+		if err := uploadProjectRepository(botClient, project + "." + smName); err != nil {
```

Comment:
This argument is meant to be the name of the remote for upload.  the argument is used to run (for the default remote name `origin` and branch `master`):
```
git push origin master
git annex copy --to=origin
```

Changing it to the repository name will fail, unless we're also changing remote names, but that doesn't seem to be part of this patch.